.github/workflows/docker.yml

```diff
@@ -28,6 +28,9 @@ defaults:
 jobs:
   docker:
     runs-on: ubuntu-latest
+    # Only run this job if branch that triggered this workflow is the master branch
+    # and the current repository owner is Kitware (don't want to run on forks)
+    if: github.ref == 'refs/heads/master' && github.repository_owner == 'Kitware'
```

Comment:
Given our discussion last week, I meant specifically that the `github.repository_owner == 'Kitware'` part conditional might be applied to the job, while I still think that the `github.ref == 'refs/heads/master'` should still only be applied to the "Push Images" step, otherwise the docker images are not being built for legitimate PRs (like this one).

I also failed to find some obvious documentation on which repository `github.repository_owner` refers to: the merge target repository or the source repository. I supose we'd find out pretty quickly if the `github.ref == 'refs/heads/master'` is moved back to the "Push" step.